3ds Max/Max2Babylon/Exporter/BabylonExporter.cs

```diff
@@ -243,11 +243,26 @@ public void ExportClosedContainers()
             foreach (IIContainerObject containerObject in sceneContainers)
             {
                 if (!containerObject.IsInherited)continue;
+                ScriptsUtilities.ExecuteMaxScriptCommand($@"(getNodeByName(""{containerObject.ContainerNode.Name}"")).LoadContainer()");
+                bool update = containerObject.UpdateContainer();
```

Comment:
3ds Max\Max2Babylon\Exporter\BabylonExporter.cs(247,47): Error CS1955: Non-invocable member 'IIContainerObject.UpdateContainer' cannot be used like a method.

Reply:
there is probably a compatibility problem because I tested only with max 2019
I'll fix it tomorrow